### PR TITLE
Kirilushkin Aleksandr Tasks 1-10

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function trimWhiteSpaces(str) {
+  let tempArr = str.split(' ');
```
Comment (alexkond) [inline tasks/10_remove-whitespaces/remove-whitespaces.js]: Consider using `string.replace()` method

File: tasks/3_is-boolean/is-boolean.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function isBoolean(value) {
+  if (typeof(value) === 'boolean') {
```
Comment (alexkond) [inline tasks/3_is-boolean/is-boolean.js]: `typeof(value) === 'boolean'` itself returns `boolean`.
This is a bad practice:
``` javascript
if(true){
 return true;
} else {
 return false;
}```

File: tasks/5_profile-lookup/profile-lookup.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+function lookupProfile(firstName, prop) {
+  for (const item of contacts) {
+    if (item.firstName === firstName) {
+      for (const key in item) {
+        if (item.hasOwnProperty(prop)) {
+          if (key === 'firstName') {
+            continue;
+          }
+          return item[prop];
```
Comment (alexkond) [inline tasks/5_profile-lookup/profile-lookup.js]: Could be simplified to 
``` javascript
if (key !== 'firstName') {
 return item[prop];
}
```

File: tasks/9_find-numbers/find-numbers.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function getNumbers(str) {
+  const regex = /\d+/g;
+  let digitsArr = str.match(regex);
+  for (let index = 0; index < digitsArr.length; index++) {
+    digitsArr[index] = +digitsArr[index];
```
Comment (alexkond) [inline tasks/9_find-numbers/find-numbers.js]: Please use `Number(val)` to cast a value to a number, it clearly shows your intent

File: tasks/7_random-fraction/random-fraction.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+function randomFraction() {
+  return +Math.random().toFixed(1);
```
Comment (alexkond) [inline tasks/7_random-fraction/random-fraction.js]: Please use `Number(val)` to cast a value to a number, it clearly shows your intent

**Review by alexkond — CHANGES_REQUESTED** [review]

**Review by akimovaa — DISMISSED** [review]

File: tasks/1_golf-score/golf-score.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+let GOLF_SCORES = {
```
Comment (KalachevDev) [inline tasks/1_golf-score/golf-score.js]: const

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+function trimWhiteSpaces(str) {
+  return str.replace(/ /g, '');
```
Comment (alexkond) [inline tasks/10_remove-whitespaces/remove-whitespaces.js]: Use `\s` symbol to match spaces in regex

**Review by alexkond — DISMISSED** [review]

File: tasks/1_golf-score/golf-score.js (hunk 6 above)
Comment (vovcyan) [inline tasks/1_golf-score/golf-score.js]: const

**Review by vovcyan — DISMISSED** [review]

**Review by KalachevDev — DISMISSED** [review]

**Review by alexkond — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

**Pull request merged**